CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# UNRELEASED
+- [NEW] Add faceted search variable argument to `drillDown` method allowing multiple drill down
+  values to be specified.
```

Comment:
maybe add `for a single field name.` since it was already possible to have multiple `drillDown` values with different field names.